tlslite/utils/cryptomath.py

```diff
@@ -253,8 +253,7 @@ def numBytes(n):
 # **************************************************************************
 
 def getRandomNumber(low, high):
-    if low >= high:
-        raise AssertionError()
+    assert low <= high
```

Comment:
This should say `low < high` to preserve the same semantics. Does it make sense to generate random, where low == high? If so, the change in semantics should be reflected in the commit message.

Reply:
good catch, fixed